src/js/angular/stats/app.js

```diff
@@ -2,57 +2,6 @@ import 'angular/core/services';
 
 const adminInfoApp = angular.module('graphdb.framework.stats', ['toastr']);
 
-adminInfoApp.config(['$routeProvider', '$locationProvider', '$menuItemsProvider',
-    function ($routeProvider, $locationProvider, $menuItemsProvider) {
-        $routeProvider.when('/sysinfo', {
-            templateUrl: 'pages/info.html',
-            controller: 'AdminInfoCtrl',
-            title: 'System information',
-            helpInfo: 'The System information view shows the configuration values of the JVM '
-            + 'running the GraphDB Workbench as well generate a detailed server report file you can use to hunt down issues'
-        }).when('/webapi', {
-            templateUrl: 'pages/webapi.html',
-            title: 'REST API documentation',
-            helpInfo: 'The REST API view documents the available public RESTful endpoints and '
-            + 'provides an interactive interface to execute the requests.'
-        });
-
-        const getDocLink = function () {
-            const pi = $menuItemsProvider.getProductInfo();
```

Comment:
I removed the relation between the menuItemsProvider and these functions and passed the productInfo directly to the menu items during the rendering.